packages/build/src/commands/run_command.js

```diff
@@ -12,6 +12,7 @@ const { firePluginCommand } = require('./plugin')
 const { getCommandReturn } = require('./return')
 
 // Run a command (shell or plugin)
+// eslint-disable-next-line max-lines-per-function
```

Comment:
I'm not happy about disabling the ESLint rule here, but this particular function is not straightforward to refactor because we're essentially passing down properties to other functions.

@ehmicky what do you think?

Reply:
This makes sense :+1: